queries/helm/highlights.scm

```diff
@@ -0,0 +1,46 @@
+; inherits: gotmpl
+
+; Builtin functions
+(function_call
+  function: (identifier) @function.builtin
+  (#any-of? @function.builtin
+    "and" "or" "not" "eq" "ne" "lt" "le" "gt" "ge" "default" "required" "empty" "fail" "coalesce"
+    "ternary" "print" "println" "printf" "trim" "trimAll" "trimPrefix" "trimSuffix" "lower" "upper"
+    "title" "untitle" "repeat" "substr" "nospace" "trunc" "abbrev" "abbrevboth" "initials"
+    "randAlphaNum" "randAlpha" "randNumeric" "randAscii" "wrap" "wrapWith" "contains" "hasPrefix"
+    "hasSuffix" "quote" "squote" "cat" "indent" "nindent" "replace" "plural" "snakecase" "camelcase"
+    "kebabcase" "swapcase" "shuffle" "toStrings" "toDecimal" "toJson" "mustToJson" "toPrettyJson"
+    "mustToPrettyJson" "toRawJson" "mustToRawJson" "fromYaml" "fromJson" "fromJsonArray"
+    "fromYamlArray" "toYaml" "regexMatch" "mustRegexMatch" "regexFindAll" "mustRegexFinDall"
+    "regexFind" "mustRegexFind" "regexReplaceAll" "mustRegexReplaceAll" "regexReplaceAllLiteral"
+    "mustRegexReplaceAllLiteral" "regexSplit" "mustRegexSplit" "sha1sum" "sha256sum" "adler32sum"
+    "htpasswd" "derivePassword" "genPrivateKey" "buildCustomCert" "genCA" "genSelfSignedCert"
+    "genSignedCert" "encryptAES" "decryptAES" "now" "ago" "date" "dateInZone" "duration"
+    "durationRound" "unixEpoch" "dateModify" "mustDateModify" "htmlDate" "htmlDateInZone" "toDate"
+    "mustToDate" "dict" "get" "set" "unset" "hasKey" "pluck" "dig" "merge" "mustMerge"
+    "mergeOverwrite" "mustMergeOverwrite" "keys" "pick" "omit" "values" "deepCopy" "mustDeepCopy"
+    "b64enc" "b64dec" "b32enc" "b32dec" "list" "first" "mustFirst" "rest" "mustRest" "last"
+    "mustLast" "initial" "mustInitial" "append" "mustAppend" "prepend" "mustPrepend" "concat"
+    "reverse" "mustReverse" "uniq" "mustUniq" "without" "mustWithout" "has" "mustHas" "compact"
+    "mustCompact" "index" "slice" "mustSlice" "until" "untilStep" "seq" "add" "add1" "sub" "div"
+    "mod" "mul" "max" "min" "len" "addf" "add1f" "subf" "divf" "mulf" "maxf" "minf" "floor" "ceil"
+    "round" "getHostByName" "base" "dir" "clean" "ext" "isAbs" "kindOf" "kindIs" "typeOf" "typeIs"
+    "typeIsLike" "deepequal" "semver" "semverCompare" "urlParse" "urlJoin" "urlquery" "lookup"
+    "include")
+  )
```

Comment:
```suggestion
    "include"))
```
The formatter messed up again.

Reply:
:facepalm: Time for me to refactor the queries again

Reply:
Not my fault but rahter neovim :D 

```scheme
(program ; [0, 0] - [5, 0]
  (comment) ; [0, 0] - [0, 19]
  (list ; [1, 0] - [4, 2029]
    (symbol) ; [1, 1] - [1, 14]
    (symbol) ; [2, 2] - [2, 11]
    (list ; [2, 12] - [2, 24]
      (symbol)) ; [2, 13] - [2, 23]
    (symbol) ; [2, 25] - [2, 42]
    (list ; [3, 2] - [4, 2028]
      (ERROR) ; [3, 3] - [3, 11]
      (symbol) ; [3, 12] - [3, 29]
      (string) ; x 200
      (string)))) ; [4, 2018] - [4, 2027]
```

Reply:
Tbh I'm more than surprised that the formatter doesn't go boom with it. So I take that as a win

Reply:
Linter has the last word. Either we wait until the upstream issue https://github.com/neovim/neovim/issues/27705 is fixed, or we merge the messed up version and reformat later (probably easier).